test/canary-load/mzcompose.py

```diff
@@ -128,11 +129,24 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
                 except FailedTestExecutionError as e:
                     assert len(e.errors) > 0, "Exception contains no errors"
                     for error in e.errors:
+                        # TODO(def-): Remove when #22576 is fixed
+                        if "Non-positive multiplicity in DistinctBy" in error.message:
+                            continue
```

Comment:
This will just suppress the printed message but the error will still get collected. Is this intended?

Reply:
Probably not when looking at the next change.

Reply:
True, fixed